src/en/clouds-LXD.md

```diff
@@ -1,30 +1,16 @@
 Title: Juju LXD local provider
 
-# Configuring for LXD
+# Using LXD as a cloud provider 
```

Comment:
I thought we were getting away from "cloud provider" terminology in favor of just "cloud"